_use_cases/encryption-at-rest.markdown

```diff
@@ -2,7 +2,9 @@
 name: Encryption At Rest
 ---
 
-## Why
+### Encryption At Rest
+
+#### Why
```

Comment:
I adjusted the nesting level to be in keeping with the overview, which i think looks nicer.